lib/CMakeLists.txt

```diff
@@ -38,8 +42,15 @@ add_library (ChakraCoreStatic STATIC
   $<TARGET_OBJECTS:Chakra.Parser>
   ${wasm_objects}
   ${wabt_objects}
+  ${lttng_objects}
 )
 
+if(USE_LTTNG)
```

Comment:
lttng is not for osx ++ we better share target_link_libs below.. So, move this under `else()` for linux below and define `${LTTNG....}` and use it under the same target_link_libraries

Reply:
Having multiple `target_link_libraries` is fine and additive so that shouldn't cause issues. Since lttng is opt-in, I was thinking that on mac people would just not specify the `--lttng` flag (or if they did and it broke, they would see that lttng wasn't supported on their system). Do you feel strongly about this?

Reply:
Could be nice to combine them in the same place in order to prevent future `what is happening here` confusions.